qa/workunits/windows/test_rbd_wnbd.py

```diff
@@ -461,31 +598,44 @@ def _get_stamp(self):
         buff += b'\0' * padding
         return buff
 
+    def _get_stamp_path(self):
+        return self.image.path
+
     @Tracer.trace
     def _write_stamp(self):
-        with open(self.image.path, 'rb+') as disk:
+        with open(self._get_stamp_path(), self._write_open_mode) as disk:
             stamp = self._get_stamp()
             disk.write(stamp)
 
     @Tracer.trace
     def _read_stamp(self):
-        with open(self.image.path, 'rb') as disk:
+        with open(self._get_stamp_path(), self._read_open_mode) as disk:
             return disk.read(len(self._get_stamp()))
 
     @Tracer.trace
     def run(self):
         # Wait up to 5 seconds and then check the disk,
         # ensuring that nobody else wrote to it.
         time.sleep(self._rand_float(0, 5))
```

Comment:
Should this sleep along with the comment be moved into `_expect_path_exists` branch as well?  Not seeing the point of waiting if the check isn't going to be performed.

Reply:
good point